ledger-core/application/functest/wallet_delete_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+// +build functest
+
+package functest
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Creates wallet, calls /wallet/delete, then calls /wallet/get_balance and gets error.
+func TestWalletDelete(t *testing.T) {
+	t.Log("C4859")
```

Comment:
I suggest skipping whole test for now and uncomment the last part

Reply:
done